lua/mini/misc.lua

```diff
@@ -44,6 +44,10 @@ end
 MiniMisc.config = {
   -- Array of fields to make global (to be used as independent variables)
   make_global = { 'put', 'put_text' },
+  cursor_goto = {
```

Comment:
Let's not add it to setup.
It is better suited as fields of `opts` argument for `setup_goto_last_cursor()` and `goto_last_cursor()`.